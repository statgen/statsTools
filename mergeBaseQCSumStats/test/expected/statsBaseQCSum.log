NonOverlapRegionPos::add: Invalid Range,  start must be < end, but 109 >= 109
NonOverlapRegionPos::add: Invalid Range,  start must be < end, but 112 >= 111
