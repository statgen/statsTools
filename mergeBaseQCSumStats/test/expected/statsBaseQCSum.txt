chrom	chromStart	chromEnd	TotalReads	Dups	QCFail	Mapped	Paired	ProperPaired	ZeroMapQual	MapQual<10	MapQual255	PassMapQual	AverageMapQuality	AverageMapQualCount	Depth	Q20Bases
1	104	105	3	0	0	3	3	0	1	2	0	3	11.000	3	3	0
1	105	106	3	0	0	3	3	0	1	2	0	3	11.000	3	3	3
1	106	107	3	0	0	3	3	0	1	2	0	3	11.000	3	3	3
2	108	109	3	0	0	3	3	0	1	2	0	3	11.000	3	0	0
2	109	110	3	0	0	3	3	0	1	2	0	3	11.000	3	0	0
2	110	111	3	0	0	3	3	0	1	2	0	3	11.000	3	0	0
2	10012	10013	42	6	6	24	21	3	8	16	0	24	11.000	24	21	0
4	107	108	3	0	0	3	3	0	1	2	0	3	11.000	3	3	3
4	10012	10013	42	6	6	24	21	3	8	16	0	24	11.000	24	21	0
4	10013	10014	39	6	6	21	18	3	7	14	0	21	11.000	21	21	15
4	10014	10015	39	6	6	21	18	3	7	14	0	21	11.000	21	21	3
